zero/kustomize/deployment/base.yaml

```
@@ -0,0 +1,16 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: pomerium
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      app.kubernetes.io/name: pomerium-zero
+  template:
+    spec:
+      automountServiceAccountToken: true
+      serviceAccountName: pomerium-zero
+      containers:
+        - name: pomerium
```

```suggestion
        - name: pomerium
          readinessProbe:
            httpGet:
              path: /healthz
              port: 80
            initialDelaySeconds: 10
            periodSeconds: 5
          livenessProbe:
            httpGet:
              path: /healthz
              port: 80
            initialDelaySeconds: 15
            periodSeconds: 10
```

Not sure this maps to true readyness /live however....